plugins/@grouparoo/postgres/src/lib/shared/getPropertyValue.ts

```diff
@@ -58,8 +59,9 @@ export const getPropertyValue: GetPropertyValueMethod<
       throw new Error(`${aggregationMethod} is not a known aggregation method`);
   }
 
-  const params = [tableName];
-  let query = `SELECT ${aggSelect} as __result FROM %I WHERE`;
+  const params: string[] = [];
+  const from = makeFromClause({ tableName, sourceQuery }, params);
```

Comment:
On the postgres side, lots of the table methods are also reused, but they now use this `makeFromClause` helper any time we need to query their source